src/main/scala/edu/cmu/cs/obsidian/Main.scala

```diff
@@ -378,9 +385,18 @@ object Main {
                         Paths.get(mainName)
                 }
                 val protobufOutputPath = outputPath.resolve("protos")
+                val temp = new File(protobufOutputPath.toString)
```

Comment:
I don't think this part is right. When I try to compile InsuranceService.obs, I get this:

```
Exception in thread "main" java.nio.file.DirectoryNotEmptyException: InsuranceService/protos
	at sun.nio.fs.UnixCopyFile.copy(UnixCopyFile.java:564)
	at sun.nio.fs.UnixFileSystemProvider.copy(UnixFileSystemProvider.java:253)
	at java.nio.file.Files.copy(Files.java:1274)
	at edu.cmu.cs.obsidian.Main$.$anonfun$compileProgram$3(Main.scala:398)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at edu.cmu.cs.obsidian.Main$.compileProgram(Main.scala:353)
	at edu.cmu.cs.obsidian.Main$.main(Main.scala:252)
	at edu.cmu.cs.obsidian.Main.main(Main.scala)
```